From d84225c276b7d31be75a6bbcfaa8b0f1722b663b Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Thu, 23 Jul 2015 18:49:05 +0800
Subject: [PATCH 0611/5242] MLK-11281-4 ARM: imx: add iomuxc lpsr gpr node for
 imx7d

commit  c0bcdba687495cd9f28d44de8dd3172a492b6eb7 from
https://source.codeaurora.org/external/imx/linux-imx.git

Add LPSR GPR node to support LPSR mode on i.MX7D.

Signed-off-by: Anson Huang <b20788@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7d.dtsi |    5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm/boot/dts/imx7d.dtsi b/arch/arm/boot/dts/imx7d.dtsi
index 8166184..3703bfb 100644
--- a/arch/arm/boot/dts/imx7d.dtsi
+++ b/arch/arm/boot/dts/imx7d.dtsi
@@ -172,6 +172,11 @@
 		mipi-phy-supply = <&reg_1p0d>;
 		pcie-phy-supply = <&reg_1p0d>;
 	};
+
+	iomuxc_lpsr_gpr: lpsr-gpr@30270000 {
+		compatible = "fsl,imx7d-lpsr-gpr";
+		reg = <0x30270000 0x10000>;
+	};
 };
 
 &anatop {
-- 
1.7.9.5

